apps/editor/test/unit/wysiwygEditor.spec.js

```diff
@@ -324,6 +324,46 @@ describe('WysiwygEditor', () => {
       expect(wwe.getValue()).toEqual(`${expectedHtml}<br />`);
     });
 
+    it('should replace space(32) to &nbsp; in the front/back b tag', () => {
+      const html = [
+        '<b>foo </b><br />',
+        '<b>foo </b><i>bar</i>',
+        '<b class="foo">bar </b><i>baz</i>',
+        '<b class="foo"> bar</b><i>baz</i>'
+      ].join('');
+
+      wwe.setValue(html);
+
+      const expectedHtml = [
+        '<b>foo&nbsp;</b><br />',
+        '<b>foo&nbsp;</b><i>bar</i>',
+        '<b class="foo">bar&nbsp;</b><i>baz</i>',
+        '<b class="foo">&nbsp;bar</b><i>baz</i>'
+      ].join('');
+
+      expect(wwe.getValue()).toEqual(`${expectedHtml}<br />`);
```

Comment:
You should use `toBe` instead of `toEqual` when comparing string values. If you're using it in other test cases, that's legacy code and needs to be changed. The test case below needs to be modified in the same way.